jovian/jovian_nb_ext/main.js

```diff
@@ -668,6 +754,18 @@ define([
     jvn_notif.element.attr("disabled", true);
   }
 
+  function dropdownOption(modal, option) {
+    return new Promise(res => {
+      modal.modal("hide");
+      let it = setInterval(() => {
+        if ($("#jvn_options").length == 0) {
+          option();
+          res(clearInterval(it));
+        }
+      }, 10);
+    });
+  }
+
```

Comment:
What is this used for?

Reply:
This function was used to avoid keyboard loss when switch from dropdown menu to other modules.

Without calling this function to connect other modules, the keyboard will always focus on Notebook cells, and can not correctly input anything to the input box of the module window.

Reply:
Keyboard Manager Parameter should take care of that.

Can you record a screen cast of the problem described in the previous message?

Reply:
Yes, I can do it by the demo